src/services/store/auth/Slices.ts

```diff
@@ -56,7 +59,10 @@ const appSlice = createSlice({
     checkDeviceError: (state, { payload }) => {
       state.devicePin = payload?.detail || '';
       if (payload.status !== 401) {
-        state.errorString = `${payload.status} - ${payload?.title}`;
+        state.errorString =
+          payload.status || payload?.title
```

Comment:
What will be if an error will have a status code and doesn`t have any title?  for example error = {status: 400, title: ""}  return string '400 - ' in our case. wi need to use payload.status && payload?.title or show defaultPinError